CODE_OF_CONDUCT.md

```
@@ -0,0 +1,65 @@
+
+### 3. Temporary Ban
+**Impact**: A serious violation of community standards, including sustained inappropriate behavior.  
+**Consequence**: A temporary ban from any form of interaction or public communication with the community.
+
+### 4. Permanent Ban
+**Impact**: Demonstrating a pattern of sustained inappropriate behavior or harassment.  
+**Consequence**: A permanent ban from the community.
+
+## Attribution
+
+This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org), version 2.1, available at https://www.contributor-covenant.org/version/2/1/code_of_conduct.html.
```

Markdownlint

Bare URL used

(MD034, no-bare-urls)